src/main/java/org/logstash/dissect/fields/NormalField.java

```diff
@@ -0,0 +1,52 @@
+package org.logstash.dissect.fields;
+
+import com.logstash.Event;
+import org.logstash.dissect.ValueResolver;
+
+import java.util.Map;
+
+public final class NormalField extends AbstractField {
+
+    private static final Field missing = new NormalField("missing field", 100000);
+    public static Field getMissing() {
```

Comment:
I'd say there is no need for a method. Just make the constant public. But you should be aware that `Field` is mutable, so somebody might innocently change something and suddenly strange values start to appear out of nowhere because everybody is getting the same instance...